src/storage/sqlite.py

```
@@ -3843,33 +3916,32 @@ def list_recent_events(
         params.append(limit)
-        # Only fixed SQL fragments are appended here; user input stays parameterized.
         with self._connection(commit=False) as connection:
             rows = connection.execute(
                 f"""
-                SELECT * FROM calendar_events
+                SELECT * FROM cal_econ_event
                 WHERE {' AND '.join(conditions)}
-                ORDER BY timestamp DESC, id DESC
+                ORDER BY datetime(event_time_utc) DESC, provider_event_id DESC
                 LIMIT ?
                 """,
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)